docs/content/modules/tensor_mechanics/systems/AuxKernels/RadialDisplacementCylinderAux.md

```diff
@@ -0,0 +1,11 @@
+<!-- MOOSE Object Documentation Stub: Remove this when content is added. -->!devel /AuxKernels/RadialDisplacementCylinderAux float=right width=auto margin=20px padding=20px background-color=#F8F8F8
+
+
+# RadialDisplacementCylinderAux
+!description /AuxKernels/RadialDisplacementCylinderAux
+
+!parameters /AuxKernels/RadialDisplacementCylinderAux
+
+!inputfiles /AuxKernels/RadialDisplacementCylinderAux
+
+!childobjects /AuxKernels/RadialDisplacementCylinderAux
```

Comment:
These classes were checked in by Ben after I started working on the documentation, so these stubs were automatically generated for those new classes.  I'll talk with Ben about these auxkernels and add documentation in a new PR.